### ISSUE TITLE
build NEL into loader

### ISSUE THREAD

**Issue by kdutia** [TheScienceMuseum/heritage-connector#243]
- [x] build capability in `datastore.NERLoader`
- [ ] ~add ability to save and load classifier from disk rather than retraining~ classifier training is fast enough since using unique indexes retrain every time. Someone who needs to can always pickle the class instance
- [x] add to `smg_loader`
- [x] batch so it uses an acceptable amount of memory 
- [x] run on EC2
- [x] check code TODOs
- [ ] ~NERLoader: add ground truth labels from training data separately (rather than predicting on training data)~ training data is very small compared to predicted data (~1000 vs 2.5M rows)
- [x] NERLoader: separately add entities without link candidates and entities with link candidates (both batched according to `batch_size`)
